parm/stage/master_gefs.yaml.j2

```
@@ -114,7 +114,7 @@
 {% endfilter %}
 {% endif %}
 
-{% if REPLAY_ICS %}
+{% if REPLAY_ICS or USE_ATM_ENS_PERTURB_FILES %}
```

Should replay just not turn on this setting?
```suggestion
{% if USE_ATM_ENS_PERTURB_FILES %}
```